src/jsonld/image.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { JsonLd, JsonLdProps } from './jsonld';
+
+type Creator = {
+  '@type': 'Person' | 'Organisation';
+  name: string;
+};
+
+interface ImageJsonLd {
+  contentUrl?: string;
+  creator?: Creator;
+  creditText?: string;
+  copyrightNotice?: string;
+  license?: string;
+  acquireLicensePage?: string;
+}
+
+export interface ImageJsonLdProps extends JsonLdProps {
+  images?: ImageJsonLd[];
```

Comment:
I wonder do we just expose the `images` prop. 

In the case that there is a single image we will handle to correct output format?

Might reduce overhead of managing to ways passing props.

Also in the case that it is API driven, we may output a single image in array.

Reply:
Think I agree with just having the one option actually.

How do you mean about API driven part? Do you mean if an array of one image is passed, we output and array of one rather than stripping the array and returning just the object?